src/liblibc/lib.rs

```diff
@@ -3955,6 +3962,10 @@ pub mod consts {
             pub const _SC_STREAM_MAX : c_int = 27;
             pub const _SC_TZNAME_MAX : c_int = 28;
             pub const _SC_PAGESIZE : c_int = 39;
+
+            // Android
```

Comment:
It's ok to remove these comments as it's already implied by the location of the definition anyway